checador/app/assets/stylesheets/companies.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the companies controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: https://sass-lang.com/
```

Comment:
If you don't occupy this file, delete it.

Reply:
Thanks, the changes were made, could you check